lb.yaml

```diff
@@ -0,0 +1,8 @@
+ndctl:
+  nvdimm-util:
+    build:
+      - make -f Makefile.lb
```

Comment:
@lital-lb this a bit cumbersome sequence and return in every repo, why didn't we added default target for build and install ?